azkaban-common/src/main/java/azkaban/trigger/builtin/SlaChecker.java

```diff
@@ -69,16 +69,23 @@ public static SlaChecker createFromJson(final HashMap<String, Object> obj)
 
   private Boolean isSlaMissed(final ExecutableFlow flow) {
     final SlaType type = slaOption.getType();
+    logger.info("SLA type for flow " + flow.toString() + " is " + type);
     if (flow.getStartTime() < 0) {
+      logger.info("Start time is less than 0 for flow " + flow.toString());
       return false;
     }
 
     Status status;
     if (type.getComponent() == SlaType.ComponentType.FLOW) {
+      logger.info("SLA type is flow.");
       if (this.checkTime < flow.getStartTime()) {
+        logger.info("checktime = " + this.checkTime);
+        logger.info("SLA duration = " + slaOption.getDuration().toMillis());
```

Comment:
Consider adding the unit for the logging, e.g. `ms`?

Reply:
Looking at line 84, I believed checkTime would be in ms only.